lib/EntryManager/tests/entry-manager-test.js

```diff
@@ -15,131 +16,217 @@ import {
+                  expect(entryManagerInteractor.entryForm.isPresent).to.be.true;
+                });
+              });
+            });
+
+            describe('item deletion prohibit modal', () => {
               beforeEach(async () => {
                 await entryManagerInteractor.detailsSection.actionsButton.click();
                 await entryManagerInteractor.detailsSection.actionsDropdown.whenDeleteLoaded();
-                await entryManagerInteractor.detailsSection.actionsDropdown.duplicate.click();
+                await entryManagerInteractor.detailsSection.actionsDropdown.delete.click();
               });
 
-              it('entry form should be presented', () => {
-                expect(entryManagerInteractor.entryForm.isPresent).to.be.true;
+              it('should not be presented', () => {
+                expect(entryManagerInteractor.isProhibitDeleteModalOpen).to.be.false;
               });
             });
           });
+        });
+      });
+    });
+  });
 
-          describe('item deletion prohibit modal', () => {
-            beforeEach(async () => {
-              await entryManagerInteractor.detailsSection.actionsButton.click();
-              await entryManagerInteractor.detailsSection.actionsDropdown.whenDeleteLoaded();
-              await entryManagerInteractor.detailsSection.actionsDropdown.delete.click();
-            });
+  describe('Test 422 error', () => {
+    const resp422 = new Response();
+    resp422.status = 422;
+    resp422.json = () => {
+      return {
+        'errors': [{
+          'message': 'Cannot create entity; name is not unique',
+          'code': 'name.duplicate'
+        }]
+      };
+    };
+    setupApplication({ scenarios: ['entry-manager-dublication-error'] });
+    beforeEach(async () => mountErr(resp422));
+
+    describe('add new item', () => {
+      beforeEach(async () => {
+        await entryManagerInteractor.newItemButton.click();
+      });
 
-            it('should not be presented', () => {
-              expect(entryManagerInteractor.isProhibitDeleteModalOpen).to.be.false;
-            });
-          });
+      describe('error callout', () => {
+        beforeEach(async () => {
+          await entryManagerInteractor.entryForm.entryFormSave();
+        });
+
+        it('callout should be shown', () => {
+          expect(entryManagerInteractor.calloutElement.isPresent).to.be.true;
+        });
+
+        it('callout should display error message', () => {
+          const { errorMessage } = entryManagerInteractor.calloutElement;
+          expect(errorMessage).to.equal('Cannot create entity; name is not unique');
+        });
+      });
+    });
+  });
+
+  describe('Test 400 error', () => {
+    const resp400 = new Response();
+    resp400.status = 400;
+    resp400.statusText = '400 Bad Request';
+
+    setupApplication({ scenarios: ['controlled-vocab-default-error'] });
+    beforeEach(async () => mountErr(resp400));
```

Comment:
Can `mountErr` be moved to [here](https://github.com/folio-org/stripes-smart-components/pull/660/files#diff-3bb23c7ac7ff4457983b8a73992d2850R215)? I just do not see the reason for two beforeEach blocks when it is possible to deal with one.

The same applies to [here](https://github.com/folio-org/stripes-smart-components/pull/660/files#diff-3bb23c7ac7ff4457983b8a73992d2850R181)?